internal/reconcilers/anchor.go

```diff
@@ -397,15 +407,31 @@ func (r *AnchorReconciler) deleteNamespace(ctx context.Context, log logr.Logger,
 func (r *AnchorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	// Maps an subnamespace to its anchor in the parent namespace.
 	nsMapFn := func(obj client.Object) []reconcile.Request {
-		if obj.GetAnnotations()[api.SubnamespaceOf] == "" {
-			return nil
+		result := make([]reconcile.Request, 0)
+		nss := &corev1.NamespaceList{}
+		err := r.List(context.Background(), nss)
```

Comment:
This will result in _every_ namespace being processed _every_ time _any_ namespace is changed, with no timeout. Is this really what you want? Note that the Forest already has all the namespaces in memory.